cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1574,6 +1574,15 @@
   </property>
 
   <property>
+    <name>cdap.ugi.cache.expiration.ms</name>
```

Comment:
The caching, however, has a side effect that, if there is update to the keytab file (without changing the keytab URI, which is immutable since it's currently store inside the namespace meta), it takes the expiry time before it will be used. But I think this use case is extremely rare.